css/styles.css

```diff
@@ -71,9 +94,28 @@ body {
     left: 0;
     width: 100vw;
     height: 100vh;
-    background-color: gray;
-    opacity: 0.5;
+    background-color: rgba(0, 0, 0, .75);
     z-index: 1;
-  }
+}
+
+.close-modal {
+    position: absolute;
+    top: 8px;
+    right: 8px;
+    color: black;
+    cursor: pointer;
+    font-size: 38px;
+    text-align: right;
+    transition: all 0.1s ease;
```

Comment:
`transition` will, as the name suggests, smoothly change a CSS property's value from one value to another. this requires there to be a starting value, and an ending value (more on [CSS transitions](https://developer.mozilla.org/en-US/docs/Web/CSS/CSS_Transitions/Using_CSS_transitions))

in this case, `all` means every single property that can be transitioned will transition. it'll happen over `0.1` seconds. `ease` is how the transition will occur, in this case, "slow start, then fast, then end slowly".

the important part is the `.close-modal:hover` css block below. `:hover` is a [pseudo-class](https://developer.mozilla.org/en-US/docs/Web/CSS/:hover) that lets you apply styles to a class _only_ when the user is hovering their mouse.

so, all this together means that if you hove your mouse over an element with the class `.close-modal`, it will transition its `color` property smoothly to the `rgb` value you see on line `113`.